### ISSUE TITLE
FW update

### ISSUE THREAD

**Issue by ZdenekSL** [trezor/trezor-suite#392]
**InVision** [here](https://projects.invisionapp.com/share/VMVJK3BDYQK#/400646852_5-1-1-Settings_Device_-_Home_-)

**Comment by ZdenekSL** [comment]
https://github.com/trezor/trezor-suite/issues/58

**Comment by mroz22** [comment]
Basic implementation in #423 I would suggest merging it and add more features gradually later

- [x] refactor notifications from wallet/ to suite/ (notification is a common thing) and use them for indicating errors resulting from fw update.
- [ ] try rolling update
- [ ] move rollout to connect
- [x] make `/firmware` route static page
- [x]  resolve what happens if device disconnects (it should not show connect device page)
- [ ] dispatch proper notifications
- [x] resolve reconnecting for T1 (not tested or tried yet)
- [x] resolve reconnecting of different device than device previously connected
- [ ] what if adblocker blocks firmware download?
...

**Comment by mroz22** [comment]
Basic implementation is done, so I am lowering its priority now

**Comment by bosomt** [comment]
exit button is not working,
when one of devices in bootloader mode im not able to access suite,
after i disconetced BL device model changed to FW updated but exit button is not working

![image](https://user-images.githubusercontent.com/31506317/74647194-899fcd00-517b-11ea-9565-bc5ead4b496c.png)

**Comment by slowbackspace** [comment]
I tried the fw update (webusb) and after reconnecting the device in bootloader mode I got stuck on a screen attached below. 🤷‍♂ (device has bootloader 2.0.3, fw 2.1.0.)

Clicking the exit button and starting the proccess again got me right to the fw update progress screen.
where it says "Your device is in bootloader mode. Reconnect it to normal mode to continue.". So I again reconnected, now in normal mode, went throught the whole proccess again and now it seems to work.

edit: No it doesn't. I am in a loop wheren it says that a partial upgrade went fine, but there is still another upgrade ahead. But it keeps installing the same version 2.1.8 all over again. (Suite says I am gonna upgrade from 2.1.8 to 2.2.0, but the device always ask to confirm I want to install version 2.1.8)

<img width="704" alt="Screenshot 2020-03-11 at 12 41 13" src="https://user-images.githubusercontent.com/6961901/76413381-9936be00-6395-11ea-932d-38a77f9fcd8c.png">
